frontend/src/pages/Admin/Components/MangeTeams/ManageTeams.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import style from "./manage-teams.module.scss";
+
+export function ManageTeams(props) {
+  const cards = [
+    {
```

Comment:
let's put these data under `test_data/teams.json` [All testing data are kept there] Pleasenote that we don't have a file named `teams.json` so please create it under `test_data` folder